packages/storybook-framework-qwik/src/preset.ts

```diff
@@ -54,7 +51,7 @@ export const viteFinal: StorybookConfig["viteFinal"] = async (
 
 export const previewAnnotations: StorybookConfig["previewAnnotations"] = (
   entry = [],
-) => [...entry, require.resolve("storybook-framework-qwik/preview.js")];
+) => [...entry, join(dirname(fileURLToPath(import.meta.url)), "../preview.js")];
```

Comment:
This change seems to make `yarn workspace qwik-app storybook` fail to load for me

Reply:
```
5:00:14 PM [vite] Pre-transform error: Failed to resolve import "/Users/benjaminkindle/Developer/Web/storybook-framework-qwik/packages/storybook-framework-qwik/preview.js" from "/virtual:/@storybook/builder-vite/vite-app.js". Does the file exist?
5:00:14 PM [vite] Internal server error: Failed to resolve import "/Users/benjaminkindle/Developer/Web/storybook-framework-qwik/packages/storybook-framework-qwik/preview.js" from "/virtual:/@storybook/builder-vite/vite-app.js". Does the file exist?
  Plugin: vite:import-analysis
  File: /virtual:/@storybook/builder-vite/vite-app.js:7:81
```